<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>450</y>
     <width>599</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>229</x>
     <y>10</y>
     <width>481</width>
     <height>431</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>0</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Подсветка синтаксиса&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>481</width>
       <height>411</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>C++</string>
      </attribute>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>104</width>
         <height>31</height>
        </rect>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="plainText">
        <string>    Препроцессор</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>104</width>
         <height>31</height>
        </rect>
       </property>
       <property name="plainText">
        <string>    Строка</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit_3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>104</width>
         <height>31</height>
        </rect>
       </property>
       <property name="plainText">
        <string>    Комментарий</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit_4">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>20</y>
         <width>104</width>
         <height>31</height>
        </rect>
       </property>
       <property name="plainText">
        <string>Числ. литерал</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit_5">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>60</y>
         <width>104</width>
         <height>31</height>
        </rect>
       </property>
       <property name="plainText">
        <string>Ключ. слово</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>75</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Сменить</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>60</y>
         <width>75</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Сменить</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>100</y>
         <width>75</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Сменить</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>20</y>
         <width>75</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Сменить</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>60</y>
         <width>75</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Сменить</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>PAS</string>
      </attribute>
      <widget class="QPlainTextEdit" name="plainTextEdit_6">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>104</width>
         <height>31</height>
        </rect>
       </property>
       <property name="plainText">
        <string>    Строка</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit_7">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>104</width>
         <height>31</height>
        </rect>
       </property>
       <property name="plainText">
        <string>    Комментарий</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit_8">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>20</y>
         <width>104</width>
         <height>31</height>
        </rect>
       </property>
       <property name="plainText">
        <string>Числ. литерал</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit_9">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>60</y>
         <width>104</width>
         <height>31</height>
        </rect>
       </property>
       <property name="plainText">
        <string>Ключ. слово</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>75</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Сменить</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>60</y>
         <width>75</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Сменить</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_8">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>20</y>
         <width>75</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Сменить</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_9">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>60</y>
         <width>75</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Сменить</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>APX</string>
      </attribute>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>0</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Настройка редактора&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>201</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
